Dependency Error AspectJ Maven Plugin Installation For com:sun:tools:jar

I would like to run the Maven project that I have on https://github.com/ihsanhaikalz/testMaven in the Eclipse 4.5 (Mars) but it gave me an error in the pom.xml as follow:

1 problem was encountered while building the effective model for
 org.codehaus.mojo:aspectj-maven-plugin:1.8 [ERROR] 
'dependencies.dependency.systemPath' for com.sun:tools:jar must specify an
 absolute path but is ${toolsjarSystemPath} @

I already installed AJDT for Eclipse 4.5 through http://download.eclipse.org/tools/ajdt/45/dev/update and when I pulled from my github the Eclipse started to download Maven plugin connectors and Maven integration for AJDT but the error is still there. For the complete question please refer to http://stackoverflow.com/questions/37994466/dependency-error-aspectj-maven-plugin-installation-for-comsuntoolsjar

--- a/pom.xml
+++ b/pom.xml
@@ -195,7 +195,23 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-plugin-plugin</artifactId>
-        <!-- use parent's configuration -->
+        <!-- use parent's managed version, but override configuration because we need 'descriptor' goal -->
+        <executions>
+          <execution>
+            <id>default-descriptor</id>
+            <goals>
+              <goal>descriptor</goal>
+            </goals>
+            <phase>process-classes</phase>
+          </execution>
+          <execution>
+            <id>help-descriptor</id>
+            <goals>
+              <goal>helpmojo</goal>
+            </goals>
+            <phase>process-classes</phase>
+          </execution>
+        </executions>
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>

